### ISSUE TITLE
Problem with mediaMerging

### ISSUE THREAD

**Issue by garygreen** [jakubpawlowicz/clean-css#645]
It doesn't merge the media queries in the below css. If the `body#dashboard .pay-tip` is moved below the last media query it works.

```css
@media (max-width: 768px) {
    body#dashboard .pay-session-container {
        padding-right: 0
    }
}
body#dashboard .pay-tip {
    padding: 10px;
}
@media (max-width: 768px) {
    body#dashboard .contract-info {
        margin-top: 20px
    }
}
```

**Desired output**

```css
@media (max-width: 768px) {
    body#dashboard .pay-session-container {
        padding-right: 0
    }

    body#dashboard .contract-info {
        margin-top: 20px
    }
}
body#dashboard .pay-tip {
    padding: 10px
}
```

**Comment by jakubpawlowicz** [comment]
:+1: We should do that!

**Comment by jakubpawlowicz** [comment]
It's just been released with clean-css 3.4. :fireworks:
